.github/actions/move-released-issues-v1/README.md

```diff
@@ -0,0 +1,61 @@
+# move-released-issues-v1
+
+This action moves issues that have been released.
+
+## Inputs
+
+| Name              | Required | Description                             | Default |
+| ----------------- | -------- | --------------------------------------- | ------- |
+| `issues-url-list` | Yes      | The list of released issues URL         | NA      |
```

Comment:
We might want to call out that this input is hard dependency from the output produced by `label-released-issues-v1`